src/github.com/matrix-org/dendrite/roomserver/types/types.go

```diff
@@ -133,6 +133,12 @@ type StateEntryList struct {
 // (On postgresql this wraps a database transaction that holds a "FOR UPDATE"
 //  lock on the row holding the latest events for the room.)
 type RoomRecentEventsUpdater interface {
+	// The latest event IDs and state in the room.
+	LatestEvents() []StateAtEventAndReference
+	// The event ID of the latest event sent in the room.
```

Comment:
When you say "sent", do you mean:
 - Sent in the room?
 - Send to the output log?

We should be explicit.